lib/pantry/handlers/intent_request_handler.rb

```diff
@@ -4,8 +4,21 @@ module Handlers
 
     class IntentRequestHandler
 
-      def process(session:)
+      attr_reader :intent
+      def process(request:, user_id:)
+          @intent = request.intent
```

Comment:
Does this need to be in an instance variable?  Doesn't look like it's being referenced from other methods.  If that's the case, you don't need the `attr_reader :intent` either.